common/src/main/proto/TransportMessages.proto

```diff
@@ -140,10 +140,16 @@ message PbPartitionLocation {
   bytes mapIdBitmap = 11;
 }
 
+message PbPartitionLocationList {
+  repeated PbPartitionLocation partitionLocations = 1;
+}
+
 message PbWorkerResource {
   repeated PbPartitionLocation primaryPartitions = 1;
```

Comment:
I think the repeated modifier means that this variable is a list and there is no need to wrap it again.